packages/vite/src/node/plugins/html.ts

```diff
@@ -910,7 +912,7 @@ export function preImportMapHook(
 }
```

Comment:
Can't link in GitHub too, but in the `preImportMapHook` it would be great to warn correctly for module preloads too. Combining the regex might make this easier.